rustowl/src/owlsp.rs

```diff
@@ -721,6 +747,15 @@ impl Backend {
     }
 }
 
+impl Drop for Backend {
+    fn drop(&mut self) {
+        let rt = tokio::runtime::Runtime::new().unwrap();
```

Comment:
```suggestion
        let rt = match tokio::runtime::Runtime::new() {
            Ok(rt) => rt,
            Err(e) => {
                log::error!("Failed to create tokio::Runtime: {:?}", e);
                return;
            }
        };
```

[imho] if for some reason creating the runtime fails (e.g., out of system resources), logging the error and returning is more helpful for users

Reply:
Adding an error message is a good idea, but the current message is not well-written.  
For example, consider something like: "failed to create async runtime for graceful shutdown".